app.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each_with_index do |book, index|
+      puts "#{index + 1}. Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each_with_index do |person, index|
+      puts "#{index + 1}. Name: #{person.name}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Enter person type (student/teacher):'
+    person_type = gets.chomp.downcase
+
+    puts 'Enter age:'
+    age = gets.chomp.to_i
+
+    if person_type == 'student'
+      puts 'Enter classroom:'
+      classroom_name = gets.chomp
+      classroom = Classroom.new(classroom_name)
+      person = Student.new(age, classroom)
+    elsif person_type == 'teacher'
+      puts 'Enter specialization:'
+      specialization = gets.chomp
+      person = Teacher.new(age, specialization)
+    else
+      puts 'Invalid person type.'
+      return
+    end
+
+    @people.push(person)
+    puts 'Person created!'
+  end
+
+  def create_book
+    puts 'Enter title:'
+    title = gets.chomp
+
+    puts 'Enter author:'
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    @books.push(book)
+    puts 'Book created!'
+  end
+
+  def create_rental
+    puts 'Select a person by their index:'
+    list_people
+    person_index = gets.chomp.to_i - 1
+
+    puts 'Select a book by its index:'
+    list_books
+    book_index = gets.chomp.to_i - 1
+
+    puts 'Enter rental date (YYYY-MM-DD):'
+    date = gets.chomp
+
+    rental = @people[person_index].add_rental(@books[book_index], date)
+    @rentals.push(rental)
+    puts 'Rental created!'
+  end
```

Comment:
- [ ] As stated in the project requirement Your console app should behave in the same way as it is in the example So when Create a rental in your Console App the user doesn't have option to identify the person it selected because it doesn't show it student/teacher and also the name is 'Unknow' So Kindly Fix the issues just like the below example console App.
- A console App for Create a Rental 
![rental](https://github.com/BenMKT/OOP-school-library/assets/93598189/91293bf2-7b91-466d-8ffc-44482d0ddf00)

- Your Console App for Create Rental
![rental](https://github.com/BenMKT/OOP-school-library/assets/93598189/9eb4c7b1-b4f2-42d7-bee2-6e868ef804ed)

Reply:
- Update `create_rental` method to resolve bug.